web/src/compose_setup.js

```diff
@@ -168,7 +169,14 @@ export function initialize() {
             const topic_name = $target.attr("data-topic-name");
 
             message_edit.with_first_message_id(stream_id, topic_name, (message_id) => {
-                message_edit.toggle_resolve_topic(message_id, topic_name, true);
+                if (!message_id) {
```

Comment:
Use a more explicit check; if a real message_id of `0` was somehow returned, this would do the wrong thing!

Reply:
Oh, I had assumed that an id of 0 isn't possible. Will fix it.

Reply:
It's not possible right now, but it's always better to write code that can be verified reading just the local code, without making many assumptions about global facts that could change in the future.